docs/main-courses/congee.md

```diff
@@ -0,0 +1,27 @@
+# Congee
+
+![congee](https://thewoksoflife.com/wp-content/uploads/2016/10/congee-recipe-2.jpg)
```

Comment:
From the picture it looks very good, but the page doesn't render when I run a preview locally. This is the structure of the pages so far:

![image](https://user-images.githubusercontent.com/28093618/139573794-092e816a-31a1-436f-ba0f-b95548efa740.png)